src/Microsoft.DotNet.Build.Tasks/Microsoft.DotNet.Build.Tasks.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this an intentional change? I think what happened here is that the encoding on the csproj was changed, lets just make sure that it is the right one